metadata.rb

```diff
@@ -31,3 +31,5 @@
 depends 'mysql_connector'
 depends 'mysql2_chef_gem', '< 2.0.0'
 depends 'postgresql'
+
+chef_version '< 13.0.0'
```

Comment:
`chef_version` method is available in Chef since v12.6.0
I'd recommend to make it backward compatible:
```ruby
chef_version '< 13.0.0' if respond_to?(:chef_version)
```